cmd/presets/presets.go

```diff
@@ -1,10 +1,14 @@
 package presets
 
 // auto generated file
+
 // GetAll get all presets
 func GetAll() map[string]map[string]string {
 	var presets = make(map[string]map[string]string)
 	presets["adonis"] = map[string]string{
+		".dockerignore": `/node_modules
```

Comment:
hmm, don't we actually need the node_modules within the built image? Not sure where this line came from.

Reply:
@fabriciojs hummm...`.dockerignore` file was always inside the preset folder. But bash scripts can't read files which names start with `.`. Now I created a go script to do it, and go can read this kind of file. To remove it from here, we should remove `.dockerignore from `adonis` folder

Reply:
@fabriciojs keep them? Remove `node_modules` from all `.dockerignore`? I guess it was a bug from `parse_presets.sh` that wasn't parsing these files.

Reply:
we want to keep it...

we. want dockerignore so on COPY it won't inject your node_modules to the container build and build from scratch... i got a bug because node_modules had different symlink and broke everything...

Reply:
@danielsuguimoto oh I see! so never mind my comment, as @dbpolito pointed this indeed is a fix! 👏🏻